From 6f40d44acbfb0021f21bd63e6c0703ba701d19c4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Gu=C3=B0ni=20M=C3=A1r=20Gilbert?= <gudni.m.g@gmail.com>
Date: Sat, 2 Nov 2024 16:03:34 +0000
Subject: [PATCH] gdbus: define MAX_INPUT for musl

This is the same solution as was done in src/shared/util.c

Upstream-Status: Submitted [https://marc.info/?l=linux-bluetooth&m=173056368428988&w=2]

Signed-off-by: Guðni Már Gilbert  <gudni.m.g@gmail.com.com>
---
 gdbus/object.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/gdbus/object.c b/gdbus/object.c
index 84f116bf1..7b0476f1a 100644
--- a/gdbus/object.c
+++ b/gdbus/object.c
@@ -20,6 +20,12 @@
 #include <dbus/dbus.h>
 
 #include "gdbus.h"
+
+/* define MAX_INPUT for musl */
+#ifndef MAX_INPUT
+#define MAX_INPUT _POSIX_MAX_INPUT
+#endif
+
 #include "src/shared/util.h"
 
 #define info(fmt...)
-- 
2.43.0

